### PR TITLE
Update At Your Own Pace

### PR DESCRIPTION
**Pull request by loicreynier**
Update the At Your Own Pace Patch, of which the recommended version is now provided by At Your Own Pace.

### CODE REVIEW

**Review by sibir-ine — CHANGES_REQUESTED** [review]
`patch3rdParty` is for patches not hosted on either mods' page (hence the "third party"), so this message should be moved to `At Your Own Pace - College of Winterhold.esp` & adjusted accordingly.

**Comment by loicreynier** [comment]
Message moved to `At Your Own Pace - College Quest Expansion Patch.esp`.

File: masterlist.yaml [hunk 1]
```diff
@@ -13983,6 +13983,10 @@ plugins:
       - 'CoW Entry Requirements.esp'
       - 'NotSoFast-MageGuild.esp'
       - 'MCWT_TheAugursAdmonishment.esp'
+    msg:
+      - <<: *patchProvided
+        subs: ['College of Winterhold - Quest Expansion']
```
Comment (sibir-ine) [inline masterlist.yaml:13988]: There should be spaces separating the square brackets & single quotes:

```suggestion
        subs: [ 'College of Winterhold - Quest Expansion' ]
```

File: masterlist.yaml [hunk 2]
```diff
@@ -13983,6 +13983,10 @@ plugins:
...
+        condition: 'active("At Your Own Pace - College of Winterhold.esp") and not active("At Your Own Pace - College Quest Expansion Patch.esp")'
```
Comment (sibir-ine) [inline masterlist.yaml:13989]: The condition needs to be adjusted as well:

```suggestion
        condition: 'active("College Of Winterhold - Quest Expansion.esp") and not active("At Your Own Pace - College Quest Expansion Patch.esp")'
```

**Review by sibir-ine — CHANGES_REQUESTED** [review]

**Comment by loicreynier** [comment]
Fixed.

**Review by sibir-ine — APPROVED** [review]
Thanks!

**Pull request merged**